src/components/map/Map.tsx

```diff
@@ -74,40 +76,40 @@ const MapIcon = ({
   return <MarkerIcon color={iconColor} size={iconSize} />;
 };
 
-const renderCluster = (cluster: TCluster) => {
-  const { clusterColor: backgroundColor, geometry, id, onPress, properties = {} } = cluster;
-  const { point_count } = properties;
+const renderCluster = ({ markers, onPress }: { markers: any[]; onPress: () => void }) => {
+  let clustersData = {
+    features: [],
+    type: 'FeatureCollection'
+  };
+
+  markers.map((item) => {
```

Comment:
you want to loop here, right, not map some value and return a new array? so please change to a loop over `markers` to update the values in `clustersData.features`